services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/client/S3AsyncClientDecorator.java

```diff
@@ -33,8 +33,6 @@
 public class S3AsyncClientDecorator {
     public static final AttributeMap.Key<MultipartConfiguration> MULTIPART_CONFIGURATION_KEY =
         new AttributeMap.Key<MultipartConfiguration>(MultipartConfiguration.class){};
-    public static final AttributeMap.Key<Boolean> MULTIPART_ENABLED_KEY =
-        new AttributeMap.Key<Boolean>(Boolean.class){};
```

Comment:
If we dont need it anymore, we should also remove it from customization.config: `"contextParamConfigKey": "S3AsyncClientDecorator.MULTIPART_CONFIGURATION_KEY"` **[ln245](https://github.com/aws/aws-sdk-java-v2/pull/4899/files#diff-d517e448d4e73d698fba58683bb9144f65ff5aa70e033cd1b19dbfbd36d101daR253)**

Reply:
Ah okay. I missed that one.

Reply:
Removed it